src/ScriptCs.Core/ReplCommands/ScriptPacksCommand.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using ScriptCs.Contracts;
+using ScriptCs.Extensions;
+
+namespace ScriptCs.ReplCommands
+{
+    public class ScriptPacksCommand : IReplCommand
+    {
+        private readonly IConsole _console;
+
+        public ScriptPacksCommand(IConsole console)
+        {
+            _console = console;
+        }
+
+        public string Description
+        {
+            get { return "Displays information about script packs available in the REPL session"; }
+        }
+
+        public string CommandName
+        {
+            get { return "scriptpacks"; }
+        }
+
+        public object Execute(IRepl repl, object[] args)
+        {
+            var packContexts = repl.ScriptPackSession.Contexts;
+            var originalColor = _console.ForegroundColor;
+            _console.ForegroundColor = ConsoleColor.Yellow;
```

Comment:
This line should be moved down to inside the conditional, just before writing out the message to say there are no script packs. (See further comment below).